include/amici/rdata.h

```diff
@@ -225,24 +225,27 @@ class ReturnData {
     /** employed order forward problem (dimension: nt) */
     std::vector<int> order;
 
-    /** computation time of forward solve [s] */
+    /** computation time of forward solve [ms] */
```

Comment:
We used to have time reporting not uniformly. At least for one of them, the unit was incorrect. Now, everything should actually be in milliseconds.
No preference, whether seconds or milliseconds are better. We just need to have it the same way for all cpu times...

Reply:
preferring seconds. but either is fine, if coherently used. good to include it already in the variable/function name.

Reply:
@paszkow proposed to use the chorno library (https://en.cppreference.com/w/cpp/chrono), which looks nicer and may be more precise than what we currently have... 
Maybe something for the future?